amplify-ui-geo-explore/README.md

```diff
@@ -102,6 +102,85 @@ App has been tested on Node.js v16.13.2 and NPM v8.1.2
 7. Fill in required fields for Amazon Cognito and Amazon Location Service in [`src/configuration.js`](src/configuration.js).
 8. Start the app: run `npm start` from the sample app location on your computer.
 
+## Deploying to Amplify
+In order to run our app in the AWS Cloud vs locally, we can deploy to AWS Amplify. Follow these steps to deploy:
```

Comment:
Maybe here we can say something like: "we can deploy to AWS Amplify using the [Amplify CLI](https://docs.amplify.aws/cli/)"